domains/aqilhensem.json

```diff
@@ -4,6 +4,6 @@
         "email": "threatedblade@outlook.com"
     },
     "record": {
-        "CNAME": "qing762.github.io"
+        "CNAME": "qing762.is-a.dev/aqil"
```

Comment:
You cant put /aqil in the CNAME record

Reply:
But I just want it to be showing the contents from that specific endpoint.

Is it a must to create another domain?